sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -120,7 +142,9 @@ function withDefaultAzureCredential() {
 
 For clients that have a default browser available and for client-side applications running in the browser, the `InteractiveBrowserCredential` provides the simplest user authentication experience. In the sample below an application authenticates a `SecretClient` from the [@azure/keyvault-secrets][secrets_client_library] using the `InteractiveBrowserCredential`.
 
-> For client side applications running in the browser, the `InteractiveBrowserCredential` is the only credential type that is supported. You will also need to configure your app registration for single-page applications. Please refer to the [Single-Page application: App registration guide](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) for more information.
+For Node.js, if a `clientId` is provided, the Azure Active Directory application will need to be configured to have a "Mobile and desktop applications" redirect endpoint. Follow our guide on [setting up Redirect URIs for Desktop apps that calls to web APIs](https://docs.microsoft.com/azure/active-directory/develop/scenario-desktop-app-registration#redirect-uris).
+
+For client side applications running in the browser, the `InteractiveBrowserCredential` is the only credential type that is supported. You will also need to configure your app registration for single-page applications. Please refer to the [Single-Page application: App registration guide](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) for more information.
```

Comment:
This is already covered in https://github.com/Azure/azure-sdk-for-js/blob/d54e2dbe7d55e19aa560c3f15abbffc462442adf/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-client-side-browser-applications. Let's update the section to have the code sample as well and provide a link from here.

The entire section under `Authenticating server side applications` is for node.js.

```suggestion
For client side applications running in the browser, the `InteractiveBrowserCredential` is the only credential type that is supported. Please see [Authenticating client side browser applications](#authenticating-client-side-browser-applications)
```